core/src/main/scripts/importer/validateData.py

```diff
@@ -3717,42 +3741,63 @@ def __init__(self, *args, **kwargs):
         """Initialize a PatientResourceValidator with the given parameters."""
         super(PatientResourceValidator, self).__init__(*args, **kwargs)
         self.patient_id_lines = {}
+        self.defined_resources = {}
 
     def checkLine(self, data):
```

Comment:
normally methods use underscores in python, but i see your following the existing convention in this file so that's fine. Just FYI